dockerhub/Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM idealista/jdk:8u181-stretch-openjdk-headless
+FROM idealista/jdk:8u212-stretch-openjdk-headless
 LABEL maintainer="idealista <labs@idealista.com>"
 
-RUN apt-get update \
+RUN rm -rf /var/lib/apt/lists/* && apt-get update \
     && apt-get install -y sudo systemd systemd-sysv
```

Comment:
You should clear cache after install to shrink image size

Reply:
Updated

Reply:
an extra layer can be avoid, do it in the same step ;)